iothub/device/tests/AmqpTransportHandlerTests.cs

```diff
@@ -2,23 +2,20 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information
 
 using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Azure.Devices.Client.Test.ConnectionString;
+using Microsoft.Azure.Devices.Client.Transport.Amqp;
 using Microsoft.VisualStudio.TestTools.UnitTesting;
 
 namespace Microsoft.Azure.Devices.Client.Test.Transport
 {
-    using System.Collections.Generic;
-    using System.Net.Sockets;
-    using System.Threading;
-    using System.Threading.Tasks;
-    using Microsoft.Azure.Devices.Client.Transport;
-    using Microsoft.Azure.Devices.Client.Test.ConnectionString;
-    using Microsoft.Azure.Devices.Client.Transport.Amqp;
-
     [TestClass]
     [TestCategory("Unit")]
     public class AmqpTransportHandlerTests
     {
-        const string DumpyConnectionString = "HostName=Do.Not.Exist;SharedAccessKeyName=AllAccessKey;DeviceId=FakeDevice;SharedAccessKey=dGVzdFN0cmluZzE=";
+        private const string DumpyConnectionString = "HostName=Do.Not.Exist;SharedAccessKeyName=AllAccessKey;DeviceId=FakeDevice;SharedAccessKey=dGVzdFN0cmluZzE=";
```

Comment:
_camelCase

Reply:
not for const, and I'm not refactoring the whole file yet.